### PR TITLE
Add support for `graphql@16`

### PR DESCRIPTION
**Pull request by mhassan1**
This PR adds support for `graphql@16`. This should be part of a semver major release.

Resolves https://github.com/IBM/openapi-to-graphql/issues/458.

### CODE REVIEW

**Comment by jajjibhai008** [comment]
@Alan-Cha  could you please review this PR? We're experiencing some dependency conflicts when using the openapi-to-graphql package along with other packages that are dependent on graphql.

**Comment by StarpTech** [comment]
It looks like the CI didn't run, and probably the branch must be updated from main.

**Comment by mhassan1** [comment]
@StarpTech The branch is already up-to-date with `master`.

**Comment by StarpTech** [comment]
Then the maintainer has to allow the CI to run.

File: packages/openapi-to-graphql/test/example_api.test.ts [hunk 1]
```diff
@@ -1982,9 +1982,9 @@ test('Option selectQueryOrMutationField', () => {
   }`
 
   // The users field should exist as a Query field
-  const promise = graphql(createdSchema, query).then((result) => {
+  const promise = graphql({ schema: createdSchema, source: query }).then((result) => {
     expect(
-      result.data['__schema'].queryType.fields.find((field) => {
+      result.data['__schema']['queryType'].fields.find((field) => {
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/test/example_api.test.ts:1987]: It seems strange to me to change `.queryType` to `['queryType']` but not `.fields` to `['fields']`. I can see that the code was not consistent to begin with. I am in the opinion that the dot notation is better for brevity.

Comment (mhassan1) [reply packages/openapi-to-graphql/test/example_api.test.ts:1987]: This change was needed to resolve `TS2339: Property 'queryType' does not exist on type 'unknown'`.

File: packages/openapi-to-graphql/test/example_api7.test.ts [hunk 2]
```diff
@@ -170,13 +170,13 @@ test('Receive data from the subscription after creating a new instance', () => {
...
     setTimeout(() => {
-      graphql(createdSchema, query2, null, null, {
+      graphql({ schema: createdSchema, source: query2, variableValues: {
         deviceInput: {
           name: `${deviceName}`,
           userName: `${userName}`,
           status: false
         }
-      })
+      } })
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/test/example_api7.test.ts:179]: This also looks a little bit strange to me as well. However, I'm just going to go ahead a approve this and move on.

**Comment by Alan-Cha** [comment]
I made the mistake of merging this in without letting the CI run. I realize it was pointed out earlier but I was too hasty. I'm sorry about that. I am quite rusty about this.

Anyway, a lot of things seem to be broken. I am trying to resolve them with #482 right now.

**Comment by StarpTech** [comment]
@Alan-Cha thank you :mechanical_arm:

**Pull request merged**